src/app/mandate/mandate.component.html

```diff
@@ -0,0 +1,146 @@
+  @if (complete) {
+    <div id="banner-thank-you"></div>
+  }
+
+<!--  @if (!complete && !noAccess && !timedOut) {-->
+<!--    <mat-spinner class="complete-spinner" color="primary"-->
+<!--                 diameter="30" aria-label="Loading donation details"></mat-spinner>-->
+<!--  }-->
+
+<!--  @if (isDataLoaded) {-->
+    <div>
+      @if (false) {
+        <p class="error">
+          Sorry, you don't have access to this page. To keep your information secure we only allow viewing this
+          page temporarily after making a donation. To see donation details please check your email for a donation
+          receipt.</p>
+        <p>If you have set a donor account password you may also
+          view your <a href="/my-account/donations">donation history</a>.
+        </p>
+      }
+      @if (false) {
+        <p class="error" aria-live="assertive">
+          Your donation was not processed immediately &ndash; it may be that there is a short delay because we're very
+          busy. Please refresh this page to see the latest status. We'll also email your receipt within a few minutes when
+          your donation is processed.
+        </p>
+      }
+      <div aria-live="polite">
+        @if (complete) {
+          <div>
+            <div class="screenreader-only">
+              Thank you! Your generous regular donation has been set up.
+            </div>
+          </div>
+        }
+      </div>
+      @if (complete) {
+        <div>
+          <div class="thank-you">
+            <h2>Thank you!</h2>
+            <p class="thank-you-text">Your generous regular donation has been set up.</p>
+          </div>
+          <hr aria-hidden="true">
+          <div class="donation-summary">
+<!--            <p><strong>Hi {{ donation.firstName }},</strong></p>-->
+            <p><strong>Your donation summary</strong></p>
+            <p>Your donation of
+              <strong class="figure">{{ mandate.amount.amountInPence + ' ' + mandate.amount.currency }}</strong>
+              is worth
+              <strong class="figure">{{ totalValue + ' ' + mandate.amount.currency }}
+              each month!</strong>
+            </p>
+            <br/>
+<!--            @if (mandate?.thankYouMessage) {-->
+<!--              <p id="charity-thank-you">{{ mandate?.thankYouMessage }}</p>-->
+<!--            }-->
+          </div>
+          <div>
+            <div class="receipt">
+              <p><strong>Your Regular Donation Schedule</strong></p>
+              <!-- Date time appears as e.g. "Sep 5, 2023, 2:54 PM" -->
+              <table>
+                <tr>
+                  <td>Active from</td>
+                  <td>{{ mandate.schedule.activeFrom | date: 'shortTime'}} {{ mandate.schedule.activeFrom | date: 'mediumDate'}}</td>
+                </tr>
+                <tr>
+                  <td>Day of month</td>
+                  <td>{{ mandate.schedule.dayOfMonth | date: 'shortTime'}} {{ mandate.schedule.dayOfMonth | date: 'mediumDate'}}</td>
```

Comment:
day of month is just a number from 1 to 28, you're showing it as a date in 1970 here which is not right of course.